index.js

```diff
@@ -227,27 +227,31 @@ function loadConfig(program) {
         ...config
       , ...parsedConfig
       , tags: process.env.LOGDNA_TAGS || config.tags
-      , flushTimer: null
       , healthcheckTimer: null
       , rescanTimer: null
       }
 
+      return getos(cb)
+    }
+  , (distro, cb) => {
+      let distroName = `${(distro.dist || distro.os).replace(/Linux| /g, '')}`
+      if (distro.release) {
+        distroName += `/${distro.release}`
+      }
+      config.UserAgent = `${pkg.name}/${pkg.version} (${distroName})`
```

Comment:
This value is out of hand, we need to reel it in.  I'm fine with multiple values, but we should simplify and just comma-delimit them rather than try to use the parens.  Ultimately, this will play nicer with the node client, which will put this value in parens.

Reply:
agreed, this is going to end up gnarly - something like 

```logger-node/x.x.x (logdna-agent/x.x.x (ubuntu/x.x.x) k8s)```

+1 for just simple comma-delimited `package/version` in the future. that would give us all the same context we need in reporting/tooling. will require support on the ingestion side.

Reply:
Fixed.